Cargo.toml

```diff
@@ -1,97 +1,15 @@
-[package]
-name = "miden-crypto"
-version = "0.14.0"
-description = "Miden Cryptographic primitives"
+[workspace]
+members = [
+    "miden-crypto"
+]
+resolver = "2"  # Use the edition 2021 dependency resolver
+
+[workspace.package]
 authors = ["miden contributors"]
-readme = "README.md"
 license = "MIT"
 repository = "https://github.com/0xPolygonMiden/crypto"
-documentation = "https://docs.rs/miden-crypto/0.14.0"
+documentation = "https://docs.rs/miden-crypto"
```

Comment:
I would move `documentation` field into the crate-specific file. I'd also keep the version number (otherwise, I believe, it will always point to the latest version).